timelock-impl/src/main/metrics/timestampleases.yml

```diff
@@ -0,0 +1,17 @@
+options:
+  javaPackage: 'com.palantir.atlasdb.timelock.timestampleases'
+
+namespaces:
+  timestampLease:
+    docs: Metrics tracking TimeLock's implementation of timestamp leases
+    metrics:
+      locksHeld:
+        docs: Number of locks held in timestamp trackers
+        type: gauge
+        tags:
+          - name
+      approximateSmallestLeased:
```

Comment:
name to make sure any operator/dev knows this is best-effort and does not over-index